src/containers/shared/utils.js

```diff
@@ -330,8 +330,8 @@ export const formatLargeNumber = (d = 0, digits = 4) => {
 }
 
 // Document: https://developers.google.com/analytics/devguides/collection/analyticsjs/
-export const analytics = (type = null, fields = {}) => {
-  // Check if GoogleAnalytics is set, type and fields are not empty, type is valid
+export const analytics = (type, fields = {}) => {
```

Comment:
I believe this will not work properly without the `type = null`. You can just add `@ts-expect-error` where needed.

Reply:
everywhere we use this function we provide a type - there's no need for a default null value.